.circleci/src/workflows/commit.yml

```diff
@@ -5,6 +5,10 @@ jobs:
   - pack_backend:
       requires:
         - build_backend
+  - build_administration
+  - pack_administration:
+      requires:
+        - build_administration
```

Comment:
Why would we want to do this on every commit?

Reply:
i mean before we had build backend that included administration, so now i just readded it as a single step. Why shouldn't we just test if the webapp builds successfully?

Reply:
Building is a step that only rarely does not succeed, but cost a lot of credits. So maybe the amout of credits we use for doing this is not worth the outcome.

Reply:
`build_administration` does not only "build" it, but also runs linter, tests, etc. So this is likely to fail quickly. We could think of extracting the actual webpack build, but I think that the webpack build does not cost toooo much either.

Reply:
extracting seems like a good option. 

Even if it is not "tooo much" costs, doing it unnecessarily for every commit sums up.

Reply:
should i also extract the gradle build for backend @ztefanie ?

Reply:
`check-backend-health` has a success rate of 99% in the last 90 days. So I think it would be fine, to only run this check on commits on main or even only on delivery.

Reply:
alright. i created a `commit_main` where i do all the build jobs (just for bayern on frontend should be fine?)
https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard/4043/workflows/0002504a-3868-4ba1-abac-70945adc5099 (~11min)
And in the normal `commit` are just checks
https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard/4040/workflows/13bbb8fb-48fb-42fa-ba80-3b62e30f7b2b (~3min)

is that fine for you @ztefanie ?